{"name": "test_signup_title", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestSignUp' object has no attribute 'driver'", "trace": "self = <signup_test.TestSignUp object at 0x000001FAE0AE7700>\n\n    @pytest.fixture(scope='function', autouse=True)\n    def signup_fixture(self):\n>       self.sign = SignUp_Page(self.driver)\nE       AttributeError: 'TestSignUp' object has no attribute 'driver'\n\ntests\\signup_test.py:18: AttributeError"}, "description": "Check the SignUp Page", "start": 1730126547779, "stop": 1730126547779, "uuid": "373bc0a1-866b-413a-99db-bf3a533d7b40", "historyId": "f3cca1b076efe53217b26973c98aadcb", "testCaseId": "f3cca1b076efe53217b26973c98aadcb", "fullName": "tests.signup_test.TestSignUp#test_signup_title", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "signup_test"}, {"name": "subSuite", "value": "TestSignUp"}, {"name": "host", "value": "MSI"}, {"name": "thread", "value": "29152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.signup_test"}]}